automated_analysis.py

```diff
@@ -173,7 +173,7 @@
             for code in cc.code_scheme.codes:
                 if code.control_code == Codes.STOP:
                     continue
-                demographic_distributions[cc.analysis_file_key][code.code_id] = 0
+                demographic_distributions[cc.analysis_file_key][code.string_value] = 0
```

Comment:
The rest of automated_analysis must have got out of sync with the latest structure on newer projects if this is causing a problem. Consider updating the rest of automated_analysis instead.

Reply:
The rest of the pipelines reference string_values and the script are up to date. What was the reason why you switched to CodeIDs?

Reply:
I think it was in COVID19-Ke-Urban that I needed to make the change, so that I could use the match values on the codes to filter the geojson for constituencies in Nairobi metro. You're right that none of the other pipelines were updated with that, so I think you're ok using string_values here and we can revisit if needed in future.